pkgs/servers/monitoring/prometheus/ipsec-exporter.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  makeWrapper,
+  buildGoModule,
+  fetchFromGitHub,
+  libreswan,
+  nixosTests,
+}:
+buildGoModule rec {
+  pname = "ipsec-exporter";
+  version = "0.4.0";
+  rev = "v${version}";
```

Comment:
why have this in this scope instead of just in the `src = fetchFromGitHub` args?